Sources/StreamChat/Repositories/SyncOperations.swift

```diff
@@ -34,7 +34,7 @@ class GetChannelIdsOperation: AsyncOperation {
 
 class GetPendingConnectionDateOperation: AsyncOperation {
     init(database: DatabaseContainer, context: SyncContext) {
-        super.init(maxRetries: 2) { [weak database] done in
+        super.init(maxRetries: 2) { [weak database] _, done in
```

Comment:
nit: that's not directly related to the PR but would be nice to extract `maxRetries` to a constant at some point as it's reused across all sync flow operations.